### PR TITLE
DatabaseSeeder updated by reservation

### PR DESCRIPTION
**Pull request by KoliushkoArtem**

### CODE REVIEW

File: src/main/java/com/manardenza/utils/DatabaseSeeder.java [hunk 1]
```diff
@@ -29,6 +33,18 @@
         return roomsSeederList;
     }
 
+    private static List<Reservation> newReservationsList(int index) {
+        List<Reservation> reservationList = new ArrayList<>();
+        for (int j = 0; j < 12; j++) {
+            reservationList.add(new Reservation(new Date(117, j, 10),
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/utils/DatabaseSeeder.java:39]: MINOR Remove this use of "Date"; it is deprecated. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: src/main/java/com/manardenza/utils/DatabaseSeeder.java [hunk 2]
```diff
@@ -29,6 +33,18 @@
...
+                    new Date(117, j, 15),
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/utils/DatabaseSeeder.java:40]: MINOR Remove this use of "Date"; it is deprecated. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Review by zozich — APPROVED** [review]
Ok

**Pull request merged**